package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "2bored2wait",
-  "version": "0.1.24",
+  "version": "0.1.25",
```

Comment:
in the future please avoid automatic version bumps on pull requests. not only is it not needed, it fucks up the automatic labeling and releasing on our end.

Reply:
Opps really sorry about that. I will make sure that doesn’t happen again!